modules/common-jpa-impl/src/main/java/org/opencastproject/job/jpa/JpaJob.java

```diff
@@ -70,7 +71,17 @@
  */
 @Entity(name = "Job")
 @Access(AccessType.FIELD)
-@Table(name = "oc_job")
+@Table(name = "oc_job", indexes = {
```

Comment:
In the script the field  `dispatchable` has a default of `1`

Reply:
Same as with `job_load`. Fixed.